src/blocks/events/index.php

```diff
@@ -34,7 +39,7 @@ function coblocks_render_coblocks_events_block( $attributes, $content ) {
 				'use_timezone_with_r_rules'     => false,
 			)
 		);
-		$ical->init_url( $attributes['externalCalendarUrl'] );
+		$ical->init_url( sanitize_url( $attributes['externalCalendarUrl'] ) );
```

Comment:
The linting system thinks you should use `esc_url_raw()` instead, since `sanitize_url` is deprecated.

Reply:
Oh yeah, you're right, it seems it is "un-deprecated". Our linting system is probably not up to date, then ;)